src/WSEvents.cpp

```diff
@@ -911,6 +915,38 @@ void WSEvents::OnSceneItemVisibilityChanged(void* param, calldata_t* data) {
 	instance->broadcastUpdate("SceneItemVisibilityChanged", fields);
 }
 
+/**
+ * An item's transfrom has been changed.
+ *
+ * @return {String} `scene-name` Name of the scene.
+ * @return {String} `item-name` Name of the item in the scene.
+ *
+ * @api events
+ * @name SourceItemChanged
+ * @category sources
+ * @since 4.5.1
```

Comment:
`@since` will be updated on new releases.

```suggestion
 * @since unreleased
```